DigitalLearningSolutions.Data/DataServices/SupervisorService.cs

```diff
@@ -216,6 +218,34 @@ public SupervisorDelegateDetail GetSupervisorDelegateDetailsById(int supervisorD
                ).FirstOrDefault();
         }
 
+        public int? ValidateDelegate(int centreId, string delegateEmail)
+        {
+            int? delegateUserId = (int?)connection.ExecuteScalar(
+                 @"SELECT da.UserID AS DelegateUserID 
+                            FROM Users u
+                            INNER JOIN DelegateAccounts da
+                            ON da.UserID = u.ID
+                            LEFT JOIN UserCentreDetails ucd
+                            ON ucd.UserID = u.ID
+                            WHERE u.PrimaryEmail = @delegateEmail
+                            AND u.Active = 1 
+                            AND da.CentreID = @centreId", new { delegateEmail, centreId });
+

```

Comment:
What does the UserCentreDetails join do here? Should you have ucd.Email = @delegateEmail in the where clause too maybe?

Reply:
I agree, I think we need 
`WHERE (u.PrimaryEmail = @delegateEmail OR ucd.Email = @delegateEmail) ...`